plugins/inputs/vsphere/vsphere_test.go

```diff
@@ -222,8 +222,7 @@ func TestMaxQuery(t *testing.T) {
 
 	// Don't run test on 32-bit machines due to bug in simulator.
 	// https://github.com/vmware/govmomi/issues/1330
-	var i int
-	if unsafe.Sizeof(i) < 8 {
+	if strconv.IntSize < 64 {
 		return
 	}
```

Comment:
The referenced issue https://github.com/vmware/govmomi/issues/1330 seems to be resolved, so maybe the check should be removed at all?